waspc/data/Generator/templates/react-app/src/test/vitest/setup.ts

```diff
@@ -1,4 +1,8 @@
-import matchers from '@testing-library/jest-dom/matchers'
-import { expect } from 'vitest'
+import { afterEach } from 'vitest'
```

Comment:
I've updated the `vitest` and all the accompanying libraries to the latest versions. I did this because there was an error with using `jsx` files from deps (in our case our SDK) which would mean we would need to bundle our SDK for `vitest` to work again.

Updating to the latest version got rid of the `jsx` imports error, but, this setup still has import errors e.g. it can't resolve the `setupFiles` properly.